.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -28,63 +24,59 @@ body:
     id: actual
     attributes:
       label: Actual outcome
-      description: Please provide the output produced by the above code, which may be a screenshot, console output, etc. 
-      placeholder: if applicable, paste the console output and images here
+      description: The output produced by the above code; e.g. a screenshot, console output, etc.
```

```suggestion
      description: Paste the output produced by the code provided above
```
therefore everything starts with an action verb

---

As above. I don't think the action verbs are necessary or add any information.